.github/workflows/github-actions.yml

```diff
@@ -0,0 +1,40 @@
+# loosely based on https://github.com/actions/starter-workflows/blob/main/ci/python-app.yml, which is MIT Licensed
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: temurin-vdr-generator ci
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@37b082107ba410260a3aaddf93122e04801ce631
```

Comment:
Can you add the version as a comment after the SHA for ease of editing (e.g `# v1.0.0`)

Reply:
yes, also changed sha to latest release for both

Reply:
(as opposed to latest commit with passing checks)